app/workers/edition_batch_link_checker.rb

```diff
@@ -0,0 +1,31 @@
+# Calls the Link Checker API to verify all links in an edition per organisation
+require "sidekiq-scheduler"
+
+class EditionBatchLinkChecker
```

Comment:
Would be good to have `Worker` the end of the class name, perhaps this should be called `CheckAllEditionsLinksWorker`.